doc/source/user_guide/make_container.rst

```diff
@@ -0,0 +1,117 @@
+.. _ref_make_container:
+
+Create your Fluent Docker container
+===================================
+
+.. warning:: You need a valid Ansys license and an Ansys account to
+   follow the steps detailed in this section.
+
+You can create your Fluent Docker container by following the steps given on this page.
+This guide will use a local Ubuntu machine to copy the needed files from the Fluent 
+installation directory to the container.
+
+
+Requirements
+============
+
+* A Linux machine, preferably with Ubuntu 18.04 or later.
+  CentOS Linux distribution is not supported anymore.
+  This machine needs to have `Docker <https://www.docker.com>`_ installed.
+
+* A valid Ansys account. Your Ansys reseller should have
+  provide you with one.
+
+* The following provided files:
+  
+  * `Dockerfile <https://github.com/ansys/pyfluent/blob/main/docker/fluent/Dockerfile>`_
+  * `copy_docker_files.py <https://github.com/ansys/pyfluent/blob/main/docker/fluent/copy_docker_files.py>`_
+
+
+Procedure
+=========
+
+Specify the Ansys installation directory
+----------------------------------------
+
+Specify the pre-installed Ansys directory in the following function of 
+`copy_docker_files.py <https://github.com/ansys/pyfluent/blob/main/docker/fluent/copy_docker_files.py>`_. 
+
+.. code:: python
+
+    copy_files(src="<path to ``ansys_inc`` directory>")
+
+Copy needed files
+-----------------
+
+Change local directory where `copy_docker_files.py <https://github.com/ansys/pyfluent/blob/main/docker/fluent/copy_docker_files.py>`_ 
```

Comment:
This would be easier to understand if written in two steps:

1. Change to the local directory where the `copy_docker_files.py <https://github.com/ansys/pyfluent/blob/main/docker/fluent/copy_docker_files.py>`_ is located.
2. Run the following script to ....

Reply:
Done. Thank you.